### PR TITLE
tests: Added backup restore from backup id example

### PR DESCRIPTION
**Pull request by Ak-sky**
### Description

Added backup restore from backup id example

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: examples/backup-restore/main.tf [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+module "resource_group" {
+  source  = "terraform-ibm-modules/resource-group/ibm"
+  version = "1.1.5"
+  # if an existing resource group is not set (null) create a new one using prefix
+  resource_group_name          = var.resource_group == null ? "${var.prefix}-resource-group" : null
+  existing_resource_group_name = var.resource_group
+}
+
+data "ibm_database_backups" "backup_database" {
+  deployment_id = var.rabbitmq_db_crn
```
[the remark this answers is not in the document]
Comment (Ak-sky) [reply examples/backup-restore/main.tf:14]: My bad, I have added the crn.

**Review by ocofaigh — APPROVED** [review]

**Comment by ocofaigh** [comment]
@shemau you happy to approve too?

**Review by shemau — APPROVED** [review]
LGTM

I wonder if there is a potential issue for the future by not specifying a RabbitMQ version on the resource. My understanding is that it tests upgrades if the default version is increased at a later date, which is valid tests, so I think good.

**Pull request merged**